core/domain/question_domain.py

```diff
@@ -945,21 +981,24 @@ class QuestionSummary(python_utils.OBJECT):
     """Domain object for Question Summary."""
 
     def __init__(
-            self, question_id, question_content,
+            self, question_id, question_content, misconception_ids,
             question_model_created_on=None, question_model_last_updated=None):
         """Constructs a Question Summary domain object.
 
         Args:
             question_id: str. The ID of the question.
             question_content: str. The static HTML of the question shown to
                 the learner.
+            misconception_ids: str. The misconception ids attached to
+                the question.
             question_model_created_on: datetime.datetime. Date and time when
                 the question model is created.
             question_model_last_updated: datetime.datetime. Date and time
                 when the question model was last updated.
         """
         self.id = question_id
         self.question_content = html_cleaner.clean(question_content)
+        self.misconception_ids = misconception_ids
```

Comment:
Will it be the same as `inapplicable_misconception_ids` of the corresponding question object?

Reply:
> @kevinlee12, Thanks for the PR, LGTM for the codeowner files, left a few comments PTAL! Also, are we going to ask users to select `inapplicable_misconception_ids`? If yes, make sure to also have the flow for the question suggestion process.
> 
> [Note: I've only reviewed codeowner files]

Thanks @DubeySandeep! Yup, the users will be using the question editor interface (misconceptions section in particular) to mark optional misconceptions as "inapplicable" as they see fit. The UI for this will be introduced as part of https://github.com/oppia/oppia/issues/10491 (see first checkbox). It will work the same way in the question suggestion flow as well.

Reply:
No, this will contain all the misconceptions tagged to the question.

Reply:
> this will contain all the misconceptions tagged

Do question objects/models have these values in any property? if yes what's the property name?

Reply:
To be clear, this field is a combination of 2 lists. It will combine values from the AnswerGroup property tagged_skill_misconception_id as well as the the Question.inapplicable_misconception_ids.

For a question objects, you can do something like this to get the misconception ids:

```python
    question.question_state_data.interaction.answer_groups
    misconception_ids = [
        answer_group.to_dict()['tagged_skill_misconception_id']
        for answer_group in answer_groups
        if answer_group.to_dict()['tagged_skill_misconception_id']]
```

Reply:
>To be clear, this field is a combination of 2 lists. It will combine values from the AnswerGroup property tagged_skill_misconception_id as well as the the Question.inapplicable_misconception_ids.

Actually this might need a doc string update

Reply:
YEah maybe add a comment in the summy class*

Reply:
Done.